meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/apalis-imx8/lmp-base.cfg

```diff
@@ -29,5 +29,7 @@ CONFIG_HUSH_PARSER=y
 CONFIG_ENV_IS_IN_FAT=y
 CONFIG_ENV_FAT_INTERFACE="mmc"
 CONFIG_ENV_FAT_DEVICE_AND_PART="0:1"
+CONFIG_USE_BOOTCOMMAND=y
+CONFIG_BOOTCOMMAND="fatload mmc 0:1 ${scriptaddr} /boot.scr; source ${scriptaddr}"
```

Comment:
AFAIR `CONFIG_BOOTCOMMAND` should stay the same, as we include boot.itb, and for non-sota builds boot.scr
still stays in the first FAT partition
```
IMAGE_BOOT_FILES_mx8qm = "imx-boot u-boot.itb hdmitxfw.bin ${KERNEL_IMAGETYPE} ${@make_dtb_boot_files(d)} boot.scr uEnv.txt"

```

Reply:
Dropped this change.